### PR TITLE
Reword closing confirmation text

### LINKED ISSUE #2147: What else confirms that the peer is also closing/draining
```
An endpoint MAY transition from the closing period to the draining period if it
can confirm that its peer is also closing or draining.  Receiving a
CONNECTION_CLOSE frame is sufficient confirmation, as is receiving a stateless
reset.
```

This doesn't seem implementable without a list of all the things that
were sufficient.

### PR DESCRIPTION
**Pull request by martinthomson**
Closes #2147.

### CODE REVIEW

File: draft-ietf-quic-transport.md [hunk 1]
```diff
@@ -2156,11 +2156,10 @@ endpoint in the draining state MUST NOT send any packets.  Retaining packet
 protection keys is unnecessary once a connection is in the draining state.
 
 An endpoint MAY transition from the closing period to the draining period if it
-can confirm that its peer is also closing or draining.  Receiving a
-CONNECTION_CLOSE frame is sufficient confirmation, as is receiving a stateless
-reset.  The draining period SHOULD end when the closing period would have ended.
-In other words, the endpoint can use the same end time, but cease retransmission
-of the closing packet.
+can receives a CONNECTION_CLOSE frame or stateless reset, both of which indicate
```
Comment (kazuho) [inline draft-ietf-quic-transport.md]: s/can receives/receives/?
